src/dm-area/components/markdown-task-checkbox.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
nit: we shall use this everywhere :D

```suggestion
import * as React from "react";
```